.travis.yml

```diff
@@ -16,11 +16,13 @@ before_install:
     - git clone --quiet --depth 1 https://github.com/minrk/travis-wheels travis-wheels
 install:
     - wget http://c90cb902a90d6a506ab3-a24171cbeaa42ce11788f0df3362e902.r46.cf5.rackcdn.com/pandoc-1.15.0.6.tar.gz && tar -xzf pandoc-1.15.0.6.tar.gz
-    - pip install -f travis-wheels/wheelhouse . coveralls codecov
+    - pip install -f travis-wheels/wheelhouse . codecov coverage
     - pip install nbconvert[execute,serve,test]
     - python -m ipykernel.kernelspec --user
 script:
-    - nosetests -w /tmp --with-coverage --cover-package nbconvert nbconvert
+    - # need to cd or coverage report broken.
+    - cd /tmp/
+    - nosetests --with-coverage --cover-package nbconvert --cover-inclusive --cover-xml nbconvert --cover-xml-file=$TRAVIS_BUILD_DIR/coverage.xml
```

Comment:
Can we move the `nbconvert` which is a regular argument to `nosetests` to either the beginning of the args (after `nosetests`) or the end? It's a bit non-obvious stuffed in the middle.

Out of interest, what's the `--cover-inclusive` flag? I haven't seen that before.